src/templates/orderIndex.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+    <script src="../static/order.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+
+</head>
+<body>
+<div class="jumbotron bg-dark"></div>
+<div class="container-fluid">
+
+    <div class="row">
+        <div class="col-sm-3"></div>
+        <div class="col-sm-4">
+
+            <form action="{{ url_for('confirm') }}" method="POST">
+                <div>
+                    <div>
+                        <h3>Delivery method</h3>
+                        <input type="radio" name="delMethod" value="DoorDelivery"><label>&NonBreakingSpace; Door delivery</label><br>
+                        <input type="radio" name="delMethod" value="Pickup"><label>&NonBreakingSpace; Pick up at restaurant</label>
+                        <p class="help-block"></p>
+                    </div>
+                    <div class="col"></div>
+                    <div class="col"></div>
+                </div>
+
+                <div class="row" id="deliveryInfo" style="display: none">
+                    <div class="col-sm-1"></div>
+                    <div class="col-sm-8 border-bottom border-top border-left border-right">
```

Comment:
"border-bottom border-top border-left border-right"
Why not just "border"?
https://getbootstrap.com/docs/4.1/utilities/borders/#additive